sycl/doc/UsersManual.md

```diff
@@ -1,63 +1,120 @@
 # Users Manual
 
-The DPC++ Compiler contains many options to generate the desired binaries for
-your application.
+This is the list of SYCL specific options supported by compiler and some
+examples.
 
-## SYCL specific command line options
+Options marked as [DEPRECATED] are going to be removed in some future updates.
+Options marked as [EXPERIMENTAL] are expected to be used only in limitted cases
+and not recommended to use in production environment.
+
+## Generic options
 
 **`-fsycl`**
 
-    General enabling option for SYCL compilation mode.  This option enables
-    offload compilation for a given target using the `-fsycl-targets` option.
-    When the `-fsycl-targets` option is not provided, the default triple is
-    `spir64-unknown-unknown-sycldevice`.
+    General enabling option for SYCL compilation and linking mode. List of
+    targets can be specified with `-fsycl-targets`. This is fundamental option
+    for any SYCL compilation. All other SYCL specific options require it.
+
+**`-fsycl-targets=<T1>[,...,<Tn>]`**
+
+    Tells the compiler to generate code for specified devices. T is a target
+    triple device name. You can specify more than one target, comma separated.
```

Comment:
I think it is important to associate the widely used "ahead of time compilation" term with this option. We mention it a lot in various other docs.
```suggestion
    Enables "ahead of time" compilation ("AOT") for given device targets.
    Tells the compiler to generate code for specified devices. `T...` is an LLVM target
    triple-based string representing a target device architecture. You can specify more than one target, comma separated.
```

Reply:
Rephrased to focus on enabling part.